pubnub-kotlin/pubnub-kotlin-api/src/commonTest/kotlin/com/pubnub/test/integration/ChannelMetadataTest.kt

```diff
@@ -29,7 +30,7 @@ class ChannelMetadataTest : BaseIntegrationTest() {
     private val type = randomString()
 
     @Test
-    fun can_set_metadata() = runTest(timeout = defaultTimeout) {
```

Comment:
Why timeout being set to default is not needed any more?

Reply:
because I noticed that instead of remembering to add it to every single method, this can be controlled by setting the `kotlinx.coroutines.test.default_timeout` system property, so for example we can give higher timeout on CI if needed